src/2020-01-02-documenting-rcpp-packages.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Documenting Rcpp functions and classes in R packages"
+author: "Artem Sokolov"
+license: GPL (>= 2)
+tags: documentation
```

Comment:
I have in the past been somewhat relunctant to add new tags with new posts as it can get somewhat sprawling.  I agree that 'documentation' is not a bad one, but it is also new.  Should we either add 'modules' or replace it with it?

Reply:
Yes, let's keep it to existing tags. `modules` and (maybe?) `basics` should be a good fit.